### PR TITLE
Add stable-fast-3d

### PR DESCRIPTION
**Pull request by aleksandr-mokrov**
CVS-151527

### CODE REVIEW

File: notebooks/stable-fast-3d/stable-fast-3d.ipynb [hunk 1]
```diff
@@ -0,0 +1,1053 @@
+{
```
Comment (eaidova) [inline notebooks/stable-fast-3d/stable-fast-3d.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    import math
</pre><p>can you please consider to move conversion and inference wrappers to file?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2388/discussion/#comment-1758581798'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='11'/>

Comment (eaidova) [inline notebooks/stable-fast-3d/stable-fast-3d.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #50.        Dinov2EmbeddingsPatched.interpolate_pos_encoding, model.image_tokenizer.model.embeddings
</pre><p> do you really need to create class for patching function. Does it not work if it is just function?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2388/discussion/#comment-1758581807'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='11'/>

Comment (eaidova) [inline notebooks/stable-fast-3d/stable-fast-3d.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #11.            rgb_cond=None,
</pre><p>default values are not used anyway as you define all inputs in example inputs, could you please remove None?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2388/discussion/#comment-1758581811'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='16'/>

Comment (eaidova) [inline notebooks/stable-fast-3d/stable-fast-3d.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #17.            kwargs = {
</pre><p>are you sure that this repacking required? starting with openvino 2024.3, openvino supports nested dictionaries as inputs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2388/discussion/#comment-1758581815'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='16'/>

Comment (eaidova) [inline notebooks/stable-fast-3d/stable-fast-3d.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    import os
</pre><p>please move gradio demo creation into gradio_helper.py</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2388/discussion/#comment-1758581817'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='29'/>

Comment (aleksandr-mokrov) [reply notebooks/stable-fast-3d/stable-fast-3d.ipynb:1]: <p>Moved</p>

Comment (aleksandr-mokrov) [reply notebooks/stable-fast-3d/stable-fast-3d.ipynb:1]: <p>Replaced</p>

Comment (aleksandr-mokrov) [reply notebooks/stable-fast-3d/stable-fast-3d.ipynb:1]: <p>Removed</p>

Comment (aleksandr-mokrov) [reply notebooks/stable-fast-3d/stable-fast-3d.ipynb:1]: <p>forward method accepts **kwargs, so it is not nested dictionaries argumenta or kwarg</p>

Comment (aleksandr-mokrov) [reply notebooks/stable-fast-3d/stable-fast-3d.ipynb:1]: <p>Moved</p>

File: notebooks/stable-fast-3d/stable-fast-3d.ipynb [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
```
Comment (eaidova) [inline notebooks/stable-fast-3d/stable-fast-3d.ipynb:1]: <p>could you please add some details about model work and difference with previous TripoSR and maybe some diagram about model workflow? (<a href="https://stability.ai/news/introducing-stable-fast-3d" target="_blank">https://stability.ai/news/introducing-stable-fast-3d</a> <a href="https://stable-fast-3d.github.io" target="_blank">https://stable-fast-3d.github.io/</a>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2388/discussion/#comment-1762246944'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='0'/>

Comment (eaidova) [inline notebooks/stable-fast-3d/stable-fast-3d.ipynb:1]: <p>please add possibility to view conversion code as comment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2388/discussion/#comment-1762246945'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='8'/>

Comment (aleksandr-mokrov) [reply notebooks/stable-fast-3d/stable-fast-3d.ipynb:1]: <p>Some model specific and comparation with TripoSR was added. Diagram and description in context of converting are in "details" in the part "Convert the model to OpenVINO IR".</p>

Comment (aleksandr-mokrov) [reply notebooks/stable-fast-3d/stable-fast-3d.ipynb:1]: <p>added</p>

**Review by eaidova — APPROVED** [review]

**Pull request merged**